packages/components/Callout/macos/Callout.swift

```diff
@@ -0,0 +1,310 @@
+import Foundation
+import AppKit
+
+@objc(RCTCalloutView)
+class CalloutView: RCTView, CalloutWindowLifeCycleDelegate {
+
+	@objc public var target: NSNumber? {
+		didSet {
+			guard let targetView = bridge?.uiManager.view(forReactTag: target) else {
+				preconditionFailure("Invalid target react tag")
+			}
+			anchorView = targetView
+			updateCalloutFrameToAnchor()
+		}
+	}
+
+	@objc public var anchorRect: NSRect = .null {
+		didSet {
+			updateCalloutFrameToAnchor()
```

Comment:
Not trying to start a flame war but honest question, is this GitHub making the tabs look enormous or are those some Xcode settings we can tone down? Does this align with other .swift files in the repo?

Reply:
There's a personal GitHub setting I think where you can set the tab width. I have it set to 4. If you prefer, I can convert all my tabs to spaces and maybe we can add some listing to do that automatically

Reply:
You can also add a .editorconfig file at the root of the repo to tell GitHub how many spaces are in a tab. I can do that too with this change haha